apps/shipments/signals.py

```diff
@@ -37,19 +48,23 @@ def shipment_event_update(sender, event, listener, **kwargs):
                                                  listener.vault_id, ShipmentVaultSerializer(listener).data)
 
         # Update LOAD contract with vault uri/hash
+        LOG.debug(f'Updating load contract with hash {signature["hash"]}.')
         listener.update_vault_hash(signature['hash'])
 
 
 @receiver(post_save, sender=Shipment, dispatch_uid='shipment_post_save')
 def shipment_post_save(sender, **kwargs):
     instance, created = kwargs["instance"], kwargs["created"]
+    LOG.debug(f'Shipment post save with sender {sender}.')
```

Comment:
The `sender` of this signal is just a `Shipment` class so this log isn't really informative. Maybe instance.id (Shipment.id) instead